apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3478,6 +3623,14 @@ static void addProviderSlice(
       // Certain types have specific systems
       if (type == C4BBOrganizationIdentifierType.NPI) {
         id.setSystem(TransformerConstants.CODING_NPI_US);
+        if (!npiOrgName.isEmpty()) {
+          provider.setName(npiOrgName.get());
+        } else {
+          provider.setName("UNKNOWN");
```

Comment:
Def thinking this string should be in the constants, since we use it in multiple places

Reply:
Done